### PR TITLE
Fixed startNode and endNode funtions

### PR DESCRIPTION
**Pull request by CapnSpek**
The two functions have been fixed to match label redesign.

### CODE REVIEW

**Comment by CapnSpek** [comment]
`get_graphid_label_id` function can soon be removed I think.

File: src/backend/utils/adt/agtype.c [hunk 1]
```diff
@@ -4820,7 +4820,7 @@ Datum column_get_datum(TupleDesc tupdesc, HeapTuple tuple, int column,
  * node or edge. The function returns a pointer to a duplicated string that
  * needs to be freed when you are finished using it.
  */
-static char *get_label_name(const char *graph_name, graphid element_graphid)
```
Comment (rafsun42) [inline src/backend/utils/adt/agtype.c:4858]: The element_label_id type should be int32.
Also, if element_graphid is unused, remove it.

**Comment by rafsun42** [comment]
Also, Zainab's PR is now merged. Please refactor this PR.

File: src/backend/utils/adt/agtype.c (hunk 1 above)
Comment (CapnSpek) [reply src/backend/utils/adt/agtype.c:4858]: Changes have been made.

element_graphid is being used in the error message
`errmsg("graphid %lu does not exist", element_graphid)));`

**Comment by CapnSpek** [comment]
Done @rafsun42

File: src/backend/utils/adt/agtype.c (hunk 1 above)
Comment (rafsun42) [reply src/backend/utils/adt/agtype.c:4858]: @CapnSpek Let's change the error message to 'label id does not exist', and remove the element_graphid parameter.

Comment (CapnSpek) [reply src/backend/utils/adt/agtype.c:4858]: @rafsun42 Done! Please merge.

**Review by rafsun42 — APPROVED** [review]

**Pull request merged**